### PR TITLE
BEI-6: Setup linting with Eslint

### PR DESCRIPTION
**Pull request by hizikex**
#### WHY THIS PR?
- To auto-check common errors and maintain consistent code style.

#### WHAT DOES THIS PR DO?
- It helps to catch common mistakes.

- It helps to maintain a consistent code style.

- It also improves code readability and maintainability.

#### HOW CAN THIS PR BE TESTED?
- Create some test JavaScript or TypeScript files in your project or use existing files and
ensure that these files contain various scenarios and coding patterns that you want to test with ESLint.

#### THINGS YOU MAY WANT THE REVIEWER TO NOTE.
- Enable ESLint style checking.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,7 +13,25 @@
...
+    "eslint-plugin-react": "^7.32.2",
+    "eslint-plugin-react-hooks": "^4.6.0",
     "ts-node-dev": "^2.0.0",
     "typescript": "^5.1.6"
+  },
+  "dependencies": {
+    "@types/sequelize": "^4.28.15",
+    "mysql2": "^3.5.1",
+    "sequelize": "^6.32.1",
+    "sequelize-typescript": "^2.1.5"
```
Comment (LordUche) [inline package.json:35]: This has nothing to do with the task

File: src/config/config.ts
Comment (LordUche) [inline src/config/config.ts:1]: This file has nothing to do with the task

**Review by LordUche — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (hizikex) [reply package.json:35]: Okay. I will remove them

**Comment by hizikex** [comment]
File not related to task and dependencies not related have being removed.

**Comment by LordUche** [comment]
@hizikex it looks like you branched out of `BEI-5-Setup-MySQL-database-with-Sequelize-ORM`. You should branch out of `staging` for this task since they are unrelated.

**Comment by hizikex** [comment]
SInce I did not branch out through staging, and I have created a branch already. Should I delete this branch and then recreate it again branching out through staging?

**Pull request closed without merging**